docs/docs/user-guide/cli-config-set.md

```diff
@@ -0,0 +1,15 @@
+# Configuring the CLI for Microsoft 365
```

Comment:
This file should be named `configuring-cli.md` to better reflect the page contents